server/server.go

```diff
@@ -524,15 +536,15 @@ func (s *Server) OpenConnector(conn storage.Connector) (Connector, error) {
 		}
 	}
 
-	connector := Connector{
+	connectorInstance := Connector{
```

Comment:
GoLand was complaining variable name was bad practice as overlaps with namespace - so I fixed while groking / unrelated to issue.